test/behaviours/GuidancePageBehaviour.scala

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package behaviours
+
+import config.AppConfig
+import org.jsoup.nodes.Document
+import play.api.Application
+import play.api.i18n.Messages
+import play.api.mvc.AnyContentAsEmpty
+import play.api.test.FakeRequest
+import utils.SpecBase
+
+trait GuidancePageBehaviour {
+
+  self: SpecBase =>
+
+  val view: Document
```

Comment:
why are we not using a Setup trait and just importing spec base normally?

Reply:
No Setup is required. Do you understand the use of this file?